v0/destinations/mp/transform.js

```diff
@@ -162,6 +163,14 @@ function getTransformedJSON(message, mappingJson) {
   }
 
   if (traits) {
+    // if address attribute is an object, we are storing the attributes in root
+    if (traits.address && isObject(traits.address)) {
+      const tkeys = Object.keys(traits.address);
+      tkeys.forEach(key => {
+        traits[key] = traits.address[key];
+      });
+      delete traits.address;
+    }
```

Comment:
In L:180 we are sending data to MP regardless of IdentifyConfig mapping

Reply:
That is expected.

Reply:
I believe this is common practice for traits/properties that we couldn't map as long as the destination allows.
@utsabc WDYT?

Reply:
Yes if the value belongs outside the mapping JSON it can be considered as an extra property for mixpanel

Reply:
Nope appending `$` should be done for reserved fields only else every property in mixpanel dashboard which are not reserved will show up with `$` in dashboard